packages/serverless/src/integrations/aws_lambda.ts

```diff
@@ -0,0 +1,248 @@
+        log_stream: setupOnceContext.awsContext.logStreamName,
+        /**
+         * process.env.AWS_REGION -  this parameters given the AWS region
+         * process.env.AWS_LAMBDA_FUNCTION_NAME - this parameter provides the AWS Lambda Function name
+         */
+        url: `https://${process.env.AWS_REGION}.console.aws.amazon.com/cloudwatch/home?region=${process.env.AWS_REGION}#logsV2:log-groups/log-group/$252Faws$252Flambda$252F${process.env.AWS_LAMBDA_FUNCTION_NAME}`,
+      });
+    }
+
+    /**
+     * This function sets tags which are displayed in Sentry Dashboard
+     * @param scope  - holds additional event information
+     * @hidden
+     */
+    function setTags(scope: Scope): void {
+      scope.setTag('runtime', `${nodeJs} ${global.process.version}`);
+      scope.setTag('transaction', setupOnceContext.awsContext.functionName);
+      scope.setTag('runtime.name', nodeJs);
+      scope.setTag('server_name', String(process.env._AWS_XRAY_DAEMON_ADDRESS));
+      scope.setTag('url', 'awslambda:///' + setupOnceContext.awsContext.functionName);
+    }
+
+    /**
+     * This function return event within hub
+     * @param eventId  - contains event Id of the event
+     * @hidden
+     */
+    function invokeEvent(eventId: String): void {
+      if (hub && hub.getIntegration(AWSLambda)) {
+        hub.withScope((scope: Scope) => {
+          scope.addEventProcessor(event => {
+            event.event_id = eventId.toString();
+            return event;
+          });
+        });
+      }
+    }
+
+    /**
+     * setting parameters in scope which will be displayed as additional data in Sentry dashboard
+     * @hidden
+     */
+    function setParameters(): void {
+      if (hub && hub.getIntegration(AWSLambda)) {
+        // setting parameters in scope which will be displayed as additional data in Sentry dashboard
+        hub.configureScope((scope: Scope) => {
+          setTags(scope);
+          // runtime
+          setAdditionalRuntimeData(scope);
+          // setting the lambda parameters
+          setAdditionalLambdaParameters(scope);
+          // setting the cloudwatch logs parameter
+          setCloudwatchLogsData(scope);
+          // setting the sys.argv parameter
+          scope.setExtra('sys.argv', process.argv);
+        });
+      }
+    }
+
+    /**
+     * unhandled and handled exception
+     * @param error  - holds the error captured in AWS Lambda Function
+     * @param id  - holds event id value
+     * @param callback  - callback function
+     */
+    rapidRuntime.prototype.postInvocationError = function(error: Error, id: string, callback: () => void): void {
+      /** return current event with event id  */
+      invokeEvent(id);
+
+      /**
+       * setting parameters in scope which will be displayed as additional data in Sentry dashboard
+       */
+      setParameters();
+
+      /** capturing the exception and re-directing it to the Sentry Dashboard */
+      hub.captureException(error);
```

Comment:
We should arrange for `flush` to be called before the Lambda Function handler returns such that we ensure events were sent.

I didn't find any use of `flush` in this PR.

https://docs.sentry.io/platforms/node/aws_lambda/

Reply:
We have used flush in the latest code